src/models/SqlOutputContentProvider.ts

```diff
@@ -150,10 +150,15 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
             });
         });
 
-        // add http post handler for setting the selection in the editor
-        this._service.addPostHandler(Interfaces.ContentType.EditorSelection, (req, res): void => {
+        // add http get handler for setting the selection in the editor
+        this._service.addHandler(Interfaces.ContentType.EditorSelection, (req, res): void => {
             let uri = req.query.uri;
-            let selection: ISelectionData = req.body;
+            let selection: ISelectionData = {
+                startLine: parseInt(req.query.startLine, 10),
```

Comment:
>10 [](start = 57, length = 2)

I assume this is the number base..if 10 is the default value I'd recommend leaving off the parameter to simplify the method call.